aries_cloudagent/protocols/issue_credential/v1_0/manager.py

```diff
@@ -893,6 +895,33 @@ async def receive_credential_ack(
 
         return cred_ex_record
 
+    async def revoke_credentials(
```

Comment:
I suggest re-naming this method to something like "set_credential_revocation_state" to be more clear on what it's doing (i.e. it's not revoking the credentials, just updating the state).

Reply:
Agree. Changed this